app/controllers/users/sessions_controller.rb

```diff
@@ -50,5 +63,11 @@ def track_authentication_attempt(email)
 
       analytics.track_event('Authentication Attempt with nonexistent user')
     end
+
+    def password
+      params.require(:user)[:password]
+    rescue ActionController::ParameterMissing
+      ''
```

Comment:
I don't think we're expected to support curl requests. Also, I don't think you can make a curl request as an authenticated user. Either way, I don't think this is a valid use case. We don't check for this scenario in any of our other forms. If anything, we would want the app to raise an error so that the person making the curl request would know they did something wrong.